internal/pkg/httpserver/server.go

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
no need for new pkg http sever please remove

Reply:
removed